extensions/nyc-food-safety/src/nyc-food-safety-search.ts

```diff
@@ -0,0 +1,234 @@
+import { showToast, Toast, getPreferenceValues, LaunchProps, showHUD, Clipboard } from "@raycast/api";
+import { exec } from "child_process";
+import { ExecException } from "child_process";
+
+// Define the Command interface for type safety
+type CommandArguments = {
+  restaurantName: string;
+};
+
+interface Preferences {
+  openaiApiKey: string;
+}
+
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`